package.json

```diff
@@ -35,11 +35,11 @@
   "peerDependencies": {
     "@edx/frontend-app-authoring": "*",
     "@edx/frontend-platform": "^8.0.3",
-    "@openedx/paragon": "^22.8.1",
     "@openedx/frontend-plugin-framework": "^1.2.1",
+    "@openedx/paragon": "^22.8.1",
     "@tanstack/react-query": "4.36.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
*Note:* After a recent rebase on Authoring MFE master, the MFE wouldn't load at all until I updated the React Version to match the Authoring MFE.